cfpb/consumerfinance.gov
# Organism css

In [base.html](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/_layouts/base.html#L314-L324), we currently add javascript based on whether a block adds it as the `js` class variable its inner `Media` class. This extends that behavior to `css` class variables, which add stylesheets to the head of a rendered document if any blocks on the page need them.

Testing:
 - pull and build
 - visit http://localhost:8000/administrative-adjudication-proceedings/administrative-adjudication-docket/3d-resorts-bluegrass/
 - should look identical to prod
 - the `on-demand/sidebar-contact-info.css` file should be requested

A follow-on PR will actually methodically move the rest of the organisms and molecules to `on-demand`
